tools/test-js-with-node

```diff
@@ -47,6 +47,7 @@ EXEMPT_FILES = make_set(
         "web/shared/src/poll_data.ts",
         "web/src/about_zulip.ts",
         "web/src/add_stream_options_popover.js",
+        "web/src/add_streams_pill.ts",
```

Comment:
Should I write a test file for this instead?

Reply:
It's not necessary unless the file is easy to write tests for, which this one may not be -- the goal is to document what we don't have tests for, not test everything.